executor/point_get.go

```diff
@@ -403,14 +403,9 @@ func (e *PointGetExecutor) get(ctx context.Context, key kv.Key) ([]byte, error)
 		// fallthrough to snapshot get.
 	}
 
-	// Global temporary table is always empty, so no need to send the request.
```

Comment:
Why the global temporary table code is modified?

Reply:
It's my fault, line 407 should be modified to `e.tblInfo.TempTableType ~= model.TempTableNone`

Reply:
If you find a bug here, it's better to file another PR for the bug.
Otherwise please reset this change ...

Reply:
Actually, there is no bug in previous code. Just because after we make `TemporaryTableSnapshotReader` can read both local and global temporary data, it can be more elegant after this modification. But it's still ok to revert it.